_posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd

```diff
@@ -0,0 +1,443 @@
+---
+title: "Neural style transfer with eager execution and Keras"
+description: |
+  Continuing our series on combining Keras with TensorFlow eager execution, we show how to implement neural style transfer in a straightforward way. Based on this easy-to-adapt example, you can easily perform style transfer on your own images. 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: http://www.rstudio.com/
+bibliography: bibliography.bib
+date: 09-09-2018
+categories:
+  - Neural style transfer
```

Comment:
We are using more pithy categories right now. I'd say you could leave this one out.